### DISCUSSION TITLE
Swipe gesture for QToggle component as click alternative (UX)

### DISCUSSION THREAD

**Discussion by endlessuniverse** [quasarframework/quasar#15885] [Ideas / Proposals]
I noticed there are users that expect the toggle to behave like a range component with two states.

**Comment by suleimanas** [comment]
It can be done easy with the Touch Swipe Directive :)

`<q-toggle v-model="value" v-touch-swipe.mouse="({direction})=>(value = direction=='right')"></q-toggle>`
